src/components/AddRepository.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import classNames from 'classnames';
+import Add from '@material-ui/icons/Add';
+import CloseIcon from '@material-ui/icons/Close';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import ErrorIcon from '@material-ui/icons/Error';
+import Grid from '@material-ui/core/Grid';
+import IconButton from '@material-ui/core/IconButton';
+import TextField from '@material-ui/core/TextField';
+import Snackbar from '@material-ui/core/Snackbar';
+import SnackbarContent from '@material-ui/core/SnackbarContent';
+import { withStyles } from '@material-ui/core/styles';
+
+
+const styles2 = (theme) => ({
+  error: {
+    backgroundColor: theme.palette.error.dark,
+  },
+  icon: {
+    fontSize: 20,
+  },
+  iconVariant: {
+    opacity: 0.9,
+    marginRight: theme.spacing.unit,
+  },
+  message: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  margin: {
+    margin: theme.spacing.unit,
+  },
+});
+
+const ErrorSnackbar = (props) => (
```

Comment:
Could specify the props required with `({ open, handleClose, ... }) =>`. Might be more clear with what's used.

Reply:
Done.